examples/javascript/index.html

```diff
@@ -1,37 +1,40 @@
 <!doctype html>
 <html>
+
 <head>
-  <title>Dropbox JavaScript SDK Examples</title>
-  <link rel="stylesheet" href="/styles.css">
+    <title>Dropbox JavaScript SDK Examples</title>
+    <link rel="stylesheet" href="/styles.css">
 </head>
+
 <body>
-  <!-- Example layout boilerplate -->
-  <header class="page-header">
-    <div class="container">
-      <nav>
-        <a href="/">
-          <h1>
-            <img src="https://cfl.dropboxstatic.com/static/images/brand/logotype_white-vflRG5Zd8.svg" class="logo" />
-            JavaScript SDK Examples
-          </h1>
-        </a>
-        <a href="https://github.com/dropbox/dropbox-sdk-js/tree/master/examples/javascript" class="view-source">View Source</a>
-      </nav>
-      <h2 class="code">
-        <a href="/">examples</a>
-      </h2>
-    </div>
-  </header>
+    <!-- Example layout boilerplate -->
+    <header class="page-header">
+        <div class="container">
+            <nav>
+                <a href="/">
+                    <h1>
+                        <img src="https://cfl.dropboxstatic.com/static/images/brand/logotype_white-vflRG5Zd8.svg" class="logo" /> JavaScript SDK Examples
+                    </h1>
+                </a>
+                <a href="https://github.com/dropbox/dropbox-sdk-js/tree/master/examples/javascript" class="view-source">View Source</a>
+            </nav>
+            <h2 class="code">
+                <a href="/">examples</a>
+            </h2>
+        </div>
+    </header>
 
-  <section class="container main">
-    <ul>
-      <li><a href="basic">Basic example</a></li>
-      <li><a href="auth">Authentication example</a></li>
-      <li><a href="team">Team example</a></li>
-      <li><a href="team-as-user">Team as user example</a></li>
-      <li><a href="upload">File upload</a></li>
-      <li><a href="download">File download</a></li>
-    </ul>
-  </section>
+    <section class="container main">
+        <ul>
+            <li><a href="basic">Basic example</a></li>
+            <li><a href="auth">Authentication example</a></li>
+            <li><a href="team">Team example</a></li>
+            <li><a href="team-as-user">Team as user example</a></li>
+            <li><a href="upload">File upload</a></li>
+            <li><a href="download">File download</a></li>
+            <li><a href="pkce-browser">PKCE Browser Example</a></li>
```

Comment:
nit: Change to lowercase for consistency with the other links: "PKCE browser example"